dist/index.html

```diff
@@ -11,16 +11,20 @@
     <form class="form-container" id="formContainer">
       <label class="input-label">Select Departure Date:</label>
       <input type="date" name="date" min:  required>
-      <label class="input-label">Trip Duration:</label>
-      <input type="text" class="form-input" required>
+      <label class="input-label">Trip Duration (days):</label>
+      <input type="text" class="form-input" id="formDuration" name="duration" required>
       <label class="input-label">Number of Travelers:</label>
-      <input type="text" class="form-input" required>
+      <input type="text" class="form-input" id="formTravelers" name="travelers" required>
       <label class="input-label">Select Destination:</label>
-      <input type="text" class="form-input" required>
+      <select name="destination" class="form-input" id="formDestinations" required>
```

Comment:
This is super cool! I didn't know that you could add a "required" attribute to inputs.